### PR TITLE
Aeyb/#135 end meeting

### PR DESCRIPTION
**Pull request by jacobatan**
**Closes:** #135

**What was done:**
- Hooked up the fe and the be together
- Router restructure

**How to test:**
- Open up `frontend` and `backend`
- Create a new meeting (make sure it is live)
- End that meeting
- Meeting should end

### CODE REVIEW

File: frontend/src/pages/home/Homepage.tsx [hunk 1]
```diff
@@ -8,57 +8,73 @@ import { useState } from 'react';
...
-        .filter(meeting => meeting.startTime <= now && meeting.finishTime >= now)
-        .map(meeting => <LiveMeeting key={meeting.id} meeting={meeting} />)
-    )
-  }
+    const [isNewMeetingOpen, setIsNewMeetingOpen] = useState(false);
+    const userContext = useUserContext();
+    const meetingContext = useMeetingContext();
+    const now = Date.now();
 
+    const renderLiveMeetings = () => {
```
Comment (TylerDonghwi) [inline frontend/src/pages/home/Homepage.tsx:16]: can we sort the meetings in the order for the upcoming and live meetings?

File: frontend/src/pages/active_meeting/ActiveMeeting.tsx [hunk 2]
```diff
@@ -1,16 +1,47 @@
...
 import UserList from './components/UserList';
 import AttendanceSelect from './components/AttendanceSelect';
-import Rating from './components/RadioGroupRating'
-import { useNavigate } from 'react-router-dom';
-
+import Rating from './components/RadioGroupRating';
+import { useNavigate, useParams } from 'react-router-dom';
+import { useUserContext } from 'src/contexts/UserContext';
+import MeetingDTO from '@shared/dtos/MeetingDTO';
+import { GetMeetingData } from '@shared/responses/MeetingResponses';
```
Comment (TylerDonghwi) [inline frontend/src/pages/active_meeting/ActiveMeeting.tsx]: declared but not used?

File: frontend/src/pages/home/components/live_meeting/LiveMeeting.tsx [hunk 3]
```diff
@@ -1,31 +1,38 @@
-import { useState, useEffect } from 'react'
+import { useState, useEffect } from 'react';
 import { useNavigate } from 'react-router-dom';
 import MeetingDTO from '@shared/dtos/MeetingDTO';
```
Comment (TylerDonghwi) [inline frontend/src/pages/home/components/live_meeting/LiveMeeting.tsx]: declared but not used?

**Review by TylerDonghwi — APPROVED** [review]
minor fixes, otherwise it works great

**Review by tlooi — APPROVED** [review]
LGTM

**Pull request merged**